website/docs/components/tag/partials/code/how-to-use.md

```diff
@@ -1,24 +1,26 @@
+## How to use this component
+
 Use tags to indicate an object's categorization, i.e., for filtering. Use a [badge](/components/badge/) instead for static metadata, status, or to indicate a new feature.
 
```

Comment:
Heather was asking for these general explanations of usage to go at the top of the "Guidelines" tab without a header above it.

(See: https://hds-website-git-br-tag-hashicorp.vercel.app/components/badge-count which I updated to follow this after her feedback.)

Reply:
good catch, thanks! take a look now and let me know if it looks more correct.

Reply:
Looks good, thanks!

You may also want to add variant/property sections as per: https://github.com/hashicorp/design-system/blob/main/wiki/Website-Editing-existing-content.md#guidelines

Reply:
Another good catch! I had these on anatomy still, so I moved them to the guidelines page to match the guidelines